media-sound/csound/csound-6.06.0.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A sound processing language and software synthesizer"
+DESCRIPTION="Csound is a user-programmable and user-extensible sound processing \
+language and software synthesizer. It is not restricted to any style of music, \
+having been used for many year in at least classical, pop. techno, and ambient."
+HOMEPAGE="https://github.com/csound/csound"
+COPYRIGHT="1991 Barry Vercoe, John ffitch"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+srcGitRev="be2e7e36946ce767329d330c5cf926def3aa5851"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="6db8221e2074d9b9acb5eb0572da4406b11c5a896098a774f120b38927be4fb4"
+SOURCE_DIR="Csound-$srcGitRev"
+PATCHES="csound-$portVersion.patch"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
```

Comment:
@dash102, adding support for secondary arch is not difficult. You may take the recipe of dvdauthor as a template ;-)